hubblestack/extmods/module_runner/audit_runner.py

```diff
@@ -275,6 +274,6 @@ def _evaluate_boolean_expression(self, boolean_expr_check_list, verbose, audit_p
                     })
                     log.error(herror)
                 except Exception as exc:
-                    log.exception(exc)
+                    log.error(exc)
```

Comment:
Make sure it prints stacktrace

Reply:
yes it will print the stacktrace in this form as well